deployments/terraform_modules/santa_api/modules/lambda/api-handler/_variables.tf

```diff
@@ -31,7 +31,7 @@ variable "endpoint" {
 variable "lambda_handler" {
   type        = string
   description = "Lambda function handler path. If left blank, this will default to a formatted handler based on the endpoint variable value"
-  default     = ""
+  default     = "bootstrap"
```

Comment:
`al2.provided` requires that the executing binary be called `bootstrap` to execute properly